src/App.vue

```diff
@@ -9,7 +9,7 @@
             <p-notice-alert group="noticeBottomLeft" position="bottom left" />
             <p-notice-alert group="noticeBottomRight" position="bottom right" />
             <p-toast-alert group="toastTopCenter" position="top center" />
-            <top-notification v-if="!$store.getters['user/hasPermission']">
+            <top-notification v-if="!$store.getters['user/hasPermission'] || $store.state.error.visibleAuthorizationError">
```

Comment:
말씀하신대로 top notification 컴포넌트에 visible을 주입받고, 다시 그 변경사항을 위에서 반양해주도록 변경이 필요해보입니다.

그 이후,

```suggestion
            <top-notification v-model="visibleAuthorizationError">
            
            ...
            visibleAuthorizationError: computed({
              get: () => 
                store.getters['user/hasPermission'] || store.state.error.visibleAuthorizationError,
                set: (val) => { store.commit('error/setVisibleAuthroziationError', val) }
            })
            ...
```

이런식으로 바뀌면 원하시는대로 동작할 것 같아요. (뇌피셜입니다...)

Reply:
이렇게 변경할 경우에 다른 페이지 (visible을 주입하지 않았던 곳)에서도 전부 props로 visible을 주입하도록 변경해야하는거겠죠..!? 흠 ㅠㅠ